packages/containers/src/SidePanel/SidePanel.test.js

```diff
@@ -19,42 +22,226 @@ describe('SidePanel', () => {
+		});
+
+		it('should define the routing onClick for each "actions" item if "path" is specified', () => {
+			state.routing.locationBeforeTransitions.pathname = '/whatever/current/path';
+			const props = mapStateToProps(state, {
+				componentId: uuid.v4(),
+				actions: [
+					{
+						whateverAction: 'prop',
+						href: 'whatever/path/to/route',
+					},
+					{
+						anotherRandomSpecificActionProp: 'other prop',
+					},
+				],
+			});
+			expect(props.actions[0]).toMatchObject({
+				href: 'whatever/path/to/route',
+				onClick: expect.anything(),
+				onClickDispatch: {
+					type: ACTION_TYPE_LINK,
+					cmf: {
+						routerPush: 'whatever/path/to/route',
+					},
+				},
+			});
+			expect(props.actions[1].onClick).toBeUndefined();
+		});
+
+		cases(
+			'should define "selected" action based on the same "menuActions" item "path"',
+			({ currentRoute, itemRoute, isMatching }) => {
+				state.routing.locationBeforeTransitions.pathname = currentRoute;
+				const actionSelectable = {
+					identity: 'The one',
+					href: itemRoute,
+				};
+
+				const props = mapStateToProps(state, {
+					componentId: uuid.v4(),
+					actions: [
+						{
+							anotherRandomSpecificActionProp: 'other prop',
+							href: 'whatever/path/to/route',
+						},
+						actionSelectable,
+					],
+				});
+
+				if (isMatching) {
+					expect(props.selected).toMatchObject(actionSelectable);
+				} else {
+					expect(props.selected).toBeUndefined();
+				}
+			},
+			[
+				{
+					name: 'Usual simple path',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual/path',
+					isMatching: true,
+				},
+				{
+					name: 'Usual simple path with trainling /',
+					currentRoute: '/a/usual/path/',
+					itemRoute: '/a/usual/path',
+					isMatching: true,
+				},
+				{
+					name: 'Usual simple path with trainling / on match',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual/path/',
+					isMatching: true,
+				},
+				{
+					name: 'Subset path starting at the beginning',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual',
+					isMatching: true,
+				},
+				{
+					name: 'Complex subset path starting at the beginning',
+					currentRoute: '/some/path/not/super/really/usual/but/which/can/occur',
+					itemRoute: '/some/path/not/super/really/usual',
+					isMatching: true,
+				},
+				{
+					name:
+						'Subset path starting at the beginning but without the exact last segment (currentRoute longer)',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual/pat',
+					isMatching: false,
+				},
+				{
+					name:
+						'Subset path starting at the beginning but without the exact last segment (currentRoute shorter)',
+					currentRoute: '/a/usual/pat',
+					itemRoute: '/a/usual/path',
+					isMatching: false,
+				},
+				{
+					name:
+						'Subset path starting at the beginning but without the last item path segment matching (currentRoute with more segments)',
```

Comment:
I don't understand the difference with the one testing `'/a/usual/path'` with `'/a/usual/pat'`